### DISCUSSION TITLE
Chrome Flags Disable Extensions.

### DISCUSSION THREAD

**Discussion by ray100-sketch** [3kh0/ext-remover#798] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can I Disable Extensions Temporarily Using Chrome Flags? (Has To be Reversible) And I have to be able to assess any website. And If no working methods, don't offer me Shimmer, or anything that will permanently disable extensions. It needs to be easily enable able. Without Signing out or anything.

**Comment by AshtonDavies** [comment, marked as answer]
There's a flag that can get you into inspect and then you can use LTBEEF. I don't remember what it is, but @Brandon421-ops told me about it.

**Reply by ray100-sketch** [reply]
If you go into the LTBEEF UI, can you re enable the extensions from there?

**Reply by AshtonDavies** [reply]
Yes, but seriously why can't people use LTMEAT Flood? It'll do the job and it's reversible.
